src/main/scala/cromwell/engine/callactor/CallActor.scala

```diff
@@ -193,6 +198,7 @@ class CallActor(key: CallKey, locallyQualifiedInputs: CallInputs, backend: Backe
     val message = executionResult match {
       case SuccessfulExecution(outputs, executionEvents, returnCode, hash, resultsClonedFrom) =>
         WorkflowActor.CallCompleted(key, outputs, executionEvents, returnCode, if (workflowDescriptor.writeToCache) Option(hash) else None, resultsClonedFrom)
+      case SuccessfulFinalCall => WorkflowActor.CallCompleted(key, Map.empty, Seq.empty, 0, None, None)
```

Comment:
I've gone back and forth on this so take it for what it is. But one thought (which I don't always agree with!) is to abstract this so you could construct a CallCompleted w/ just the key and in that case it fills in all of these placeholders for you.